### ISSUE TITLE
v-navbar-item not working in v-navbar-items

### ISSUE THREAD

**Issue by ellis2323** [vuetifyjs/vuetify#66]
Hello,

I'm trying to play with the v-navbar-items components. When i use the v-bind:items, it works
but when i try to explicitly use v-navbar-item there is nothing. And the VueJs console shows 
me an empty array.

In your example, there are many:
<v-navbar-logo>Navbar</v-navbar>
I suspect it is:
<v-navbar-logo>Navbar</v-navbar-logo>

Note sure where i need to open a doc ticket...

**Comment by johnleider** [comment]
Please follow the <a href="https://github.com/vuetifyjs/vuetify/blob/master/.github/CONTRIBUTING.md">guidelines</a> on how to report an issue. In particular, provide an example on <a href="https://jsfiddle.net/">www.jsfiddle.net</a> (or a similar service) that reproduces the problem. If necessary, create a repository for us to clone with a minimal reproduction. repositories of actual projects will generally not be accepted .

Thank you.

**Comment by ellis2323** [comment]
Hello,

just do:

```
vue init vuetifyjs/webpack vuetifyjs
cd vuetifyjs
npm install
npm run dev
```

Look the result: in the navbar, there is only "navbar written"

But if you do: 
```
vue init vuetifyjs/webpack vuetifyjs
cd vuetifyjs
yarn install
npm run dev
```
Look the result: in the navbar, there is only "navbar and Get started written"

**Comment by ellis2323** [comment]
If you remove the yarn.lock and do a yarn install, it doesn't work anymore. So there's a problem with dependency.

**Comment by ellis2323** [comment]
The problem is the same with https://github.com/vuetifyjs/webpack template.

**Comment by johnleider** [comment]
Thank you. I'm omw home from work and will square it away asap.

**Comment by johnleider** [comment]
Fixed with 661b76ab8f3475ad4e6d6d034642669317cf2258.
